### PR TITLE
Make revocation tests use CRLBuilder instead of its own copy

### PR DESCRIPTION
**Pull request by bartonjs**
Some test cleanup that got missed in the CRLBuilder PR

### CODE REVIEW

File: src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs [hunk 1]
```diff
@@ -1060,79 +1060,6 @@ public static void CheckIntermediateWithExpiredRevocation(PkiOptions pkiOptions)
                 });
         }
 
-        [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/31249", PlatformSupport.AppleCrypto)]
-        public static void TestRevocationWithNoNextUpdate_NotRevoked()
```
Comment (vcsjones) [inline src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs:1065]: I'm a little apprehensive about deleting this test. While your commit message makes sense, we have seen this in the wild: https://github.com/dotnet/runtime/issues/40708.

Comment (bartonjs) [reply src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs:1065]: Hm.  I guess I can refactor it to use AsnWriter when OmitNextUpdate is true and use the new type for the 99.9% of other tests.

**Comment by bartonjs** [comment]
I learned not to ctrl+click multiple commits and then choose revert.  That only chooses the last one you selected and if it's not the top you get to merge conflict with yourself.

So I just squashed everything down to the one commit and force pushed.

<sub>Dear GitHub, I don't know what you're doing when you update the merge status grid/block/whatever, but quit moving my focus to a different textbox.</sub>

**Comment by vcsjones** [comment]
A tip, in case it wasn't obvious (because it's not), "forced-pushed" is a link:

F7FDC6F3-EEE3-40D1-9A0C-28BC8F7A1DC9

So you can see a diff between the force push references.

> Dear GitHub

Noted 🙂

**Review by vcsjones — APPROVED** [review]

**Pull request merged**